app/views/manage_assessments/coverages/new.html.erb

```diff
@@ -18,6 +18,7 @@
 
   <%= form.simple_fields_for :outcome_coverages do |fields| %>
     <%= fields.input :outcome_id,
+      selected: set_selected_attribute_with_outcome_id_if_present(@outcome),
```

Comment:
It almost works. The problem is that `selected` needs an `id` to pre-populate the field in the `new` form. If I only pass `@outcome`, the controller successfully redirects to the form but without the pre-populated outcome value. If I add `selected: @outcome.id`, the tests error saying `undefined method id for nil:NilClass`. I wrote the helper to get past this problem.